server/src/test/java/com/lunchtime/service/impl/PersonServiceImplTest.java

```diff
@@ -31,6 +31,38 @@ public void setUp() {
         registerPerson.setPassword("123456Qwerty");
     }
 
+    @Test
+    public void return_null_if_phone_number_equals_password() throws Exception {
+        Person person = Person
+            .builder()
+            .phoneNumber("+380501234567")
+            .password("+380501234567")
+            .build();
+
+        String phoneNumber = person.getPhoneNumber();
+        String password = person.getPassword();
+        if (phoneNumber.equals(password)) {
+            person = null;
+        }
+        Assert.assertNull(person);
```

Comment:
You should test your method in service, not local variable

You can check if your test works. Comment lines 26-29 in PersonServiceImpl.java, your tests will be green